client/src/styles/hostSetup.module.css

```diff
@@ -0,0 +1,26 @@
+.main {
+    background-color: var(--theme-box-background);
+    padding: var(--theme-md);
+}
+.title {
+    border-bottom: var(--theme-title-text) solid 1px;
+    margin-bottom: var(--theme-md);
+    padding-bottom: var(--theme-xs);
+    font-size: 18px;
+}
+.grid {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: var(--theme-sm) var(--theme-md);
+    align-items: end;
```

Comment:
This causes the checkbox to be aligned at the bottom of its grid row

Reply:
Grrrr